en_US.json

```diff
@@ -2065,6 +2065,8 @@
 	"security.settings.2fa.text": "2FA adds an extra layer of protection of your accounts. Each time users log in, users will need their password and a unique verification code from an authenticator app.",
 	"security.settings.2fa.title": "Two-factor Authentication",
 	"security.settings.title": "Security Settings",
+	"security.settings.update.success.text": "Security Settings has been updated",
```

Comment:
"has" should be "have" since the subject 'settings' is plural'

Reply:
Thanks for the correction. 

In the context of the portal page, I considered "Security Settings" as a collective noun (and not as a plural). So, I used "has".

Have fixed to use "have" instead of "has".